app/views/components/docs/save_this_page.yml

```diff
@@ -0,0 +1,25 @@
+name: Save this page
+description: Allows users to save a page on GOV.UK
+body: Will not render without a `page_path`
+accessibility_criteria: |
+  Buttons in this component must:
+
+  * be keyboard focusable
+  * inform the user about which item they operate on
+  * preserve focus after interacting with them
+shared_accessibility_criteria:
+- link
+examples:
+  default:
+    description: Displays the signed out user view by default
+    data:
+      page_path: 'super-nice-page-on-govuk'
+  signed_in_and_has_not_saved_page:
+    data:
+      page_path: 'super-nice-page-on-govuk'
```

Comment:
Trying to get my head around these ones 🤔 
These are just placeholder right?

Let me know if you want a more real-to-life page path example?

Reply:
yep @huwd  they're just placeholders for the examples in the [component guide](https://government-f-save-page--6earw7.herokuapp.com/component-guide/save_this_page)

Reply:
Replaced the silly dummy example with a real path from the site.